File: day008/day008.cs
using System;
class Day008 {
  static void Main() {
    int i, x, sum = 0;
    
    Console.Write("Bir sayı giriniz: ");
    x = Convert.ToInt32(Console.ReadLine());
    
    for (i = 0; i < x; i += 1) {
        Console.WriteLine(i);
        sum += i;
    }
    
    Console.WriteLine("Girdiğiniz sayıya kadar olan sayıların toplamı: " + sum);
  }
}

File: day008/day008.c
#include <stdio.h>

int main()
{
    int i, x, sum = 0;
    
    printf("Bir sayı giriniz: ");
    scanf("%d", &x);
    
    for (i = 0; i < x; i += 1) {
        printf("%d\n", i);
        sum += i;
    }
    
    printf("Girdiğiniz sayıya kadar olan sayıların toplamı: %d", sum);
    
    return 0;
}
